docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -52,32 +53,106 @@ include::_attributes.adoc[] <3>
-<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal. 
+<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal.
 <4> Set at least one category to ensure that the content is findable on the link:https://quarkus.io/guides/[Quarkus documentation home page]. For a list of Quarkus categories, see xref:{doc-guides}/doc-reference.adoc#document-attributes-and-variables[Document attributes and variables] on the "Quarkus style and content guidelines" page.
 +
 [IMPORTANT]
 ====
 Ensure there are no line breaks in the header section until after `:categories:` line.
 ====
+
+. Add an abstract that describes the purpose of the guide.
 +
-. Add an abstract to describe the purpose of the guide.
 [IMPORTANT]
 ====
-The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage]. 
+The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage].
 There must also be a line break before and after the abstract.
 ====
 
+. For how-to and tutorial topics, include a xref:prerequisites[prerequisites] section. For example:
++
+[source,asciidoc]
+----
+.Prerequisites
+
+:prerequisites-time: 30 minutes
+include::{includes}/prerequisites.adoc[]
+----
+
 For more information about the minimum header requirements, see xref:{doc-guides}/doc-reference.adoc#document-structure[Document structure] on the "Quarkus style and content guidelines" page.
 
+== Adding prerequisites
+
+For how-to and tutorial topics, include a prerequisites section above the procedure.
+
+Declaring prerequisites makes using procedures faster and easier. Include them even thought they might seem obvious to knowledgeable users.
+
+.An example prerequisite with callout explanations
+
+[source,asciidoc]
+----
+.Prerequisites <1>
+
+:prerequisites-time: 30 minutes <2>
+include::{includes}/prerequisites.adoc[] <3>
+* asdfda <4>
+
+.Procedure <5>
+----
+<1> Section heading for the prerequisites
+<2> Optional: An attribute that modifies the prerequisites
+<3> An include statement for the `prerequisites.adoc` file
+<4> Optional: An additional prerequisite not covered by the attributes
+<5> Section heading for the procedure
+
+.The default prerequisites
+
+By default, including `prerequisites.adoc` displays the following information:
+
+[quote]
+include::{includes}/prerequisites.adoc[]
+
+
+.Attributes that modify the prerequisites
```

Comment:
Super job explaining how prereqs work in the docs @rolfedh :+1: 

Wondering whether this section should be added to the [contributing reference topic ](https://quarkus.io/guides/doc-reference)along with all the attributes that determine how content gets rendered :thinking:

Reply:
Good question. For now, I'd like to keep them here, where they are more likely to be reviewed and used. 
We might eventually move them there, though.